README.md

```diff
@@ -29,18 +29,18 @@ base Julia. They contain a pointer to an object of the corresponding
 type, as well as a reference to it parent object to protect it from
 garbage collection.
 
-| Arb  | Julia  |
-|----- |------- |
-| `mag_t`  | `Mag` or `MagRef` |
-| `arf_t`  | `Arf` or `ArfRef`  |
-| `arb_t`  | `Arb` or `ArbRef`  |
-| `acb_t`  | `Acb` or `AcbRef` |
-| `arb_t*`  | `ArbVector` or `ArbRefVector` |
-| `acb_t* ` | `AcbVector` or `AcbRefVector` |
+| Arb          | Julia                         |
+| ------------ | ----------------------------- |
+| `mag_t`      | `Mag` or `MagRef`             |
+| `arf_t`      | `Arf` or `ArfRef`             |
+| `arb_t`      | `Arb` or `ArbRef`             |
+| `acb_t`      | `Acb` or `AcbRef`             |
+| `arb_t*`     | `ArbVector` or `ArbRefVector` |
+| `acb_t* `    | `AcbVector` or `AcbRefVector` |
 | `arb_mat_t`  | `ArbMatrix` or `ArbRefMatrix` |
-| `acb_mat_t`  | `AcbMatrix` or `AcbRefMatrix`  |
-| `arb_poly_t`  | `ArbPoly` or `ArbSeries` |
-| `acb_poly_t`  | `AcbPoly` or `AcbSeries` |
+| `acb_mat_t`  | `AcbMatrix` or `AcbRefMatrix` |
+| `arb_poly_t` | `ArbPoly` or `ArbSeries`      |
+| `acb_poly_t` | `AcbPoly` or `AcbSeries`      |
```

Comment:
Would it be possible to revert this change? Since it's not part of the threading.